packages/docusaurus-theme-classic/src/theme/Navbar/Layout/index.tsx

```diff
@@ -33,9 +34,15 @@ export default function NavbarLayout({children}: Props): JSX.Element {
   } = useThemeConfig();
   const mobileSidebar = useNavbarMobileSidebar();
   const {navbarRef, isNavbarVisible} = useHideableNavbar(hideOnScroll);
+  const ariaLabel = translate({
+    id: 'theme.NavBar.mainAriaLabel',
+    message: 'Main',
+    description: 'The ARIA label for the main navigation',
+  });
   return (
     <nav
       ref={navbarRef}
+      aria-label={ariaLabel}
```

Comment:
you can probably inline the translate call here directly, no need to create an intermediate variable

Reply:
Ok sweet, sounds good!  I have adjusted all the translations and removed the intermediate variable.